internal/store/postgres/migrations/20230327060200_resource_name_namespace_index.up.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX resource_name_namespace_index ON resource (name, namespace_id);
```

Comment:
Maybe put `namespace_id` instead will give more clarity?

```suggestion
CREATE INDEX resource_name_namespace_id_index ON resource (name, namespace_id);
```

Reply:
fixed